rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -2734,10 +2743,28 @@ RMW_Connext_Client::send_request(
     return RMW_RET_ERROR;
   }
 
+#ifndef TRACETOOLS_DISABLED
```

Comment:
this is the only place where this preprocessor stays, this is because the following `if statement` needs to be there always only for `TRACETOOLS_TRACEPOINT` macro, right?

Reply:
yeah, it's for the `if`. We don't need to `#ifndef TRACETOOLS_DISABLED` _for_ `TRACETOOLS_TRACEPOINT()` itself because it already expands to nothing `#ifdef TRACETOOLS_DISABLED`, but it's simpler this way.